src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -290,7 +290,8 @@
             "ActiveModeThreshold",
             "RegisteredClients",
             "ICDCounter",
-            "ClientsSupportedPerFabric"
+            "ClientsSupportedPerFabric",
+            "MaximumCheckInBackOff"
```

Comment:
`MaximumCheckInBackOff` is tagged as `apiMaturity="provisional"` - should it be here?

Reply:
Where is this set? I'm not familiar with the `apiMaturity` tag

Reply:
@ https://github.com/project-chip/connectedhomeip/blob/73ff58ff14c91c590c679a40c9570e434dbdd88f/src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml#L4

Reply:
> MaximumCheckInBackOff is tagged as apiMaturity="provisional" - should it be here?

@gmarcosb apiMaturity has absolutely nothing to do with whether things are listed in the JSON metadata.